Tools/Bluetooth/conn_sweep.py

```diff
@@ -427,89 +320,115 @@ def run_script_reset_board(sh_file):
                 hciSlave.sendAclFunc(Namespace(packetLen=str(packetLen), numPackets=str(1)))
                 hciMaster.sendAclFunc(Namespace(packetLen=str(packetLen), numPackets=str(1)))
                 hciSlave.listenFunc(Namespace(time=1, stats="False"))
+            
+            start_secs = time.time()
+
+            print('\n-----------------------------------------------------------------------------------------')
+            print(f'{args.chip} - packetLen: {packetLen}, phy: {phy}, atten: {atten}, txPower: {txPower}, testing point: {testing}')
+            print('-------------------------------------------------------------------------------------------')
+
+            print(f"\nSet the requested attenuation: {atten}.")
+            if rf_switch:
+                set_val = atten + float(args.loss)
+                mini_RCDAT = mini_RCDAT_USB(Namespace(atten=set_val))
+            
+            sleep(0.1)
+
+            print("\nReset the packet stats.")
+            hciSlave.cmdFunc(Namespace(cmd="0102FF00"), timeout=10.0)
+            hciMaster.cmdFunc(Namespace(cmd="0102FF00"), timeout=10.0)
+
+            print("\nSlave listenFunc")
+            hciSlave.listenFunc(Namespace(time=1, stats="False"))
```

Comment:
I think we can remove these two listen calls. I'm pretty sure the intended effect is to clear out any pending events before checking the packet statistics.

Reply:
Since we're doing this on lines 351 and 352, I think these 2 are redundant.

Reply:
Okay, I'll remove them. Is there a easy way to get all the HCI commands used by CMW?

Reply:
Not that I can think of. We can use the TRACE output while it's being tested to see what commands are sent. It's something along the lines of these sequences. 

```
HCI Reset
TxTest
EndTest
HCI Reset

HCI Reset
RxTest
EndTest
HCI Reset
```